tests/_graphical_wait_login.pm

```diff
@@ -61,6 +65,10 @@ sub run {
         # GDM 3.24.1 dumps a cursor in the middle of the screen here...
         mouse_hide;
         if (get_var("DESKTOP") eq 'gnome') {
+            if ($version eq '9.0') {
```

Comment:
Maybe change to everything above incl. 9 (might be harder to implement because it's string compare right now and has to change to long number compare though), we might stick with this if too much work